cmd/gaia/cmd/gaiacli/main.go

```diff
@@ -79,6 +97,30 @@ func main() {
 	}
 }
 
+// registerRoutes registers the routes from the different modules for the LCD.
+// NOTE: details on the routes added for each module are in the module documentation
+// NOTE: If making updates here you also need to update the test helper in client/lcd/test_helper.go
+func registerRoutes(rs lcd.RestServer) {
```

Comment:
Can we DRY this with `registerRoutes` in `client/lcd/test_helpers.go`? Seems like it could be a public function.

Reply:
So where would we define this? Having it in `cmd/gaia/cmd/gaiacli` indicates that this `RegisterRoutes` is gaia specific, which it is. I'll leave it there.

Reply:
bamboozled by Golang package structure again, never mind then